src/common/layout/styles.ts

```diff
@@ -0,0 +1,105 @@
+import styled from 'styled-components';
+import { getImageUrl } from '@utils/getImageUrl';
+
+export const MenuBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  position: fixed;
+  background-color: #fff1d9;
+  border-right: 7px solid #ffe8c7;
+  height: 100%;
+`;
+
+export const HeaderBox = styled.header`
+  display: flex;
+  justify-content: flex-end;
+  gap: 20px;
+  align-items: center;
+  width: 294px;
+  height: 42px;
+  position: fixed;
+  padding-right: 20px;
+  z-index: 1;
+`;
+
+export const LogoBox = styled.div`
+  margin-top: 26px;
+  margin-left: 39px;
+`;
+
+export const OverRay = styled.div`
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  z-index: 100;
+`;
+
+export const SortContainer = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+`;
+
+export const SortSelectButton = styled.button<{ $isToggled: boolean }>`
+  width: 136px;
+  height: 30px;
+  color: #fff3c0;
+  font-size: 12px;
+  font-weight: 700;
+  text-align: center;
+  background: ${({ $isToggled }) =>
+    $isToggled
+      ? `#d37744 url(${getImageUrl(
+          '정렬-아래-화살표.svg'
+        )}) no-repeat right 10px center`
+      : `#d37744 url(${getImageUrl(
+          '정렬-위-화살표.svg'
+        )}) no-repeat right 10px center`};
+  border-radius: ${({ $isToggled }) => ($isToggled ? '15px 15px 0 0' : '15px')};
```

Comment:
사실 어떤 코드를 사용하더라도 더러울 것 같긴한데, 이미 그래도 한 번만 값을 주는게 좋을 것 같아서 바꿔봤습니다.
```suggestion
  ${({ $isToggled }) =>
    css`
      background: ${
        $isToggled
          ? `#d37744 url(${getImageUrl(
              '정렬-아래-화살표.svg'
            )}) no-repeat right 10px center`
          : `#d37744 url(${getImageUrl(
              '정렬-위-화살표.svg'
            )}) no-repeat right 10px center`
      }
      border-radius: ${$isToggled ? '15px 15px 0 0' : '15px'};
    `}
```

Reply:
스타일 파일 이름만 변경한 부분이라 저기까지 확인을 못했었네요. 감사합니다 ㅎㅎ